INSTALL.md

```diff
@@ -0,0 +1,291 @@
+
+Reading `libreehr/sites/default/config.php` is a good idea.
+
+To create custom encounter forms online documentation at [LibreHealth](http://librehealth.io/). Many forms exist in `interface/forms` and may be used as examples.
+
+Other configuration settings are stored under `includes/config.php`. Everything should work out of the installation without touching those, but if  you want fax integration you will need to adjust some parameters in that file.
+
+General-purpose fax support requires customization within LibreHealthEHR at Administration->Globals and custom/faxcover.txt; it also requires the following utilities:
+
+* faxstat and sendfax from the HylaFAX client package
+* mogrify from the ImageMagick package
+* tiff2pdf, tiffcp and tiffsplit from the libtiff-tools package
+* enscript
+
+
+##   Setting Up Access Control
+
+Since LibreHealthEHR version `1.01`, phpGACL access controls are installed and configured automatically during LibreHealthEHR setup.  It can be administered within LibreHealthEHR in the admin->acl menu.  This is very powerful access control software. 
+
+Learn more about phpGACL [here](http://phpgacl.sourceforge.net/), recommend reading the phpGACL manual, the `/libreehr/Documentation/README.phpgacl` file, and the online documentation at [LibreHealth](http://librehealth.io/) . Also recommend reading the comments in `/libreehr/library/acl.inc`.
+
+##  Upgrading
+
+Be sure to back up your LibreHealthEHR installation and database before upgrading!
+
+Upgrading LibreHealthEHR is currently done by replacing the old `libreehr` directory with a newer version. And, ensure you copy your settings from the following old `libreehr` files to the new configuration files (we do not recommend simply copying the entire files):
+
+```
+libreehr/sites/default/config.php
+libreehr/sites/default/sqlconf.php
+```
+
+In this `sqlconf.php` file, set  `$config = 1;` (found near bottom of file within bunch of slashes)
+
+The following directories should be copied from the old version to the new version:
+```
+libreehr/sites/default/documents
+libreehr/sites/default/era
+libreehr/sites/default/edi
+libreehr/sites/default/letter_templates
+```
+
+If there are other files that you have customized, then you will also need to treat those carefully.
+
+To upgrade the database, run the `sql_upgrade.php` script from your web browser (for example: `http://libreehr.location/sql_upgrade.php`).  It will prompt you to select the old release number, and will display the SQL commands issued as the upgrade occurs.
```

Comment:
Do we want to keep this upgrade reference or delete it and replace it later once we decide on how we are going to do it?